ext/dom/tests/DOM4_DOMNode_before.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+DOMNode::before()
+--SKIPIF--
+<?php require_once('skipif.inc'); ?>
+--FILE--
+<?php
+require_once("dom_test.inc");
+
+$dom = new DOMDocument;
+$dom->loadXML('<test><mark>first</mark></test>');
```

Comment:
Same here, there should be another element before to see the insertion position.

Reply:
Added and realized its actually loosing nodes, need to fix the function.

Reply:
Fixed